solver/src/main/java/org/chocosolver/solver/search/measure/IMeasures.java

```diff
@@ -147,8 +159,9 @@ default String toOneLineString() {
         if (hasObjective()) {
             st.append(getBoundsManager()).append(", ");
         }
-        st.append(String.format("Resolution time %.3fs, %d Nodes (%,.1f n/s), %d Backtracks, %d Backjumps, %d Fails, %d Restarts",
+        st.append(String.format("Resolution time %.3fs, Time to first/best solution %.3fs, %d Nodes (%,.1f n/s), %d Backtracks, %d Backjumps, %d Fails, %d Restarts",
```

Comment:
Now, that I think about it, I wonder why, in satisfaction, the time of the first solution is recorded.
Could we simply ignore this value in sat?

Reply:
I wanted to adapt the text between first and best but there is no way (with the current state of code) to access if it is a satisfaction or optimisation problem inside IMeasures.

That said, I join you in the case of satisfaction, recording this value is not useful. But we fall once again in the problem of IMeasures not able to access to the status of the problem (satisfaction or optimisation).

Any idea ?

Reply:
`IMeasures#hasObjective()` is what you are looking for ;)
See for instance what is done in `IMeasures.toCSV()` (l.254)